src/bridge/mgmtscripts/add_users.py

```diff
@@ -36,6 +36,7 @@
     sys.exit(1)
 
 userdata = {
+    'id' : 1,
```

Comment:
@madflojo I actually need it for code simplicity, since `add_users.py` is only automation, used for testing purposes.

the user id is needed in `add_reactions.py` to create a sample reaction.